gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -258,17 +258,25 @@ class {{service.name}}RestTransport(_Base{{ service.name }}RestTransport):
             resp = self._interceptor.post_{{ method.name|snake_case }}(resp)
             response_metadata = [(k, str(v)) for k, v in response.headers.items()]
             resp, _ = self._interceptor.post_{{ method.name|snake_case }}_with_metadata(resp, response_metadata)
-            {# TODO(https://github.com/googleapis/gapic-generator-python/issues/2279): Add logging support for rest streaming. #}
-            {% if not method.server_streaming %}
             if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(logging.DEBUG):  # pragma: NO COVER
+                {# Logging of the streaming response payload is in `google-api-core` because the response #}
```

Comment:
Depending how we want to log (a) receiving a streaming response vs (b) exposing the `next` streamed item to the user, we could possibly want to log something here
(a) should always happen in api-core
(b) could happen in api-core, or it could happen here when we iterate to the `next` streamed item that was previously received.

Ditto for async

Reply:
I filed https://github.com/googleapis/gapic-generator-python/issues/2389 to follow up on this and I also captured this comment in the code.

Reply:
Updated comments in https://github.com/googleapis/gapic-generator-python/pull/2340/commits/58d6a89f9dee5522f59b7df271ace97cc4adde0d